cognite_toolkit/_cdf_tk/loaders/_data_loaders.py

```diff
@@ -78,7 +79,14 @@ class FileLoader(DataLoader):
     folder_name = "files"
     kind = "File"
     filetypes = frozenset()
-    exclude_filetypes = frozenset({"yml", "yaml"})
+    if Flags.REQUIRE_KIND.is_enabled():
+        exclude_filetype: frozenset[str] = frozenset({})
+        filename_pattern = (
+            # Exclude FileMetadata and CogniteFile
+            r"(?i)^(?!.*(?:FileMetadata|CogniteFile)$).*$"
+        )
+    else:
```

Comment:
Just an idea: would it make life easier if we required files to be in a subfolder called `data/`? same for other resource types like raw; 

```
raw/
    my.Table.yaml
    data/
        my.csv

```